package com.mistar.memo.domain.exception.auth

class UserNotFoundException : RuntimeException("존재하지 않는 사용자입니다.")